src/tcgwars/logic/impl/gen7/UnifiedMinds.groovy

```diff
@@ -1179,10 +1179,10 @@ public enum UnifiedMinds implements LogicCardInfo {
             text "This attack does 20 damage for each Basculin you have in play to 1 of your opponent's Benched Pokémon. (Don't apply Weakness and Resistance for Benched Pokémon.)"
             energyCost C
             attackRequirement {
-              assert my.bench.findAll({ it.name == "Basculin" }) : "You have no Basculin on your bench."
+              assertMyAll(info: "that are Basculin", { it.name == "Basculin" })
```

Comment:
bug detected + opponent needs to have a bench

Reply:
Which would be the bug here? Wouldn't `checkedArea.any(areaFilter)` return false if checkedArea is empty?
Will push the opp.bench check on the repo directly.

Reply:
When this attack is used by another pokemon (copied), let's say "Mew", does the attack look for "Basculin" or "Mew"? 

If "Basculin", then `assertMyAll(info: "that are Basculin", { it.name == "Basculin" })` should stay. Else, removed. 

What I meant by "bug" was adding a `assertOppBench()` call

Reply:
Yeah, in this case it searches specifically for "Basculin". No replicating mon, no Basculin-EX/GX/V. Similar to how Durant and now Falinks work (tho Falinks only looks for Falinks in the name, counting Falinks V)